### PR TITLE
Button

### PR DESCRIPTION
**Pull request by yash9263**
Add button and its variant styles and button docs

### CODE REVIEW

File: styles/pages/docs.css [hunk 1]
```diff
@@ -27,7 +27,6 @@
   padding: 24px 16px;
```
Comment (hsnice16) [inline styles/pages/docs.css:27]: As, Vikram suggested, you can use rems for value greater than 10px.

**Review by hsnice16 — COMMENTED** [review]
The UI is awesome man, Great work.

**Comment by Dev-AkshatGupta** [comment]
Fan of your nav bar Your UI is looking amazing and the code is clean and simple readable too
You are ready to merge it 👍

File: styles/pages/docs.css (hunk 1 above)
Comment (yash9263) [reply styles/pages/docs.css:27]: I am sticking to the multiples of 8 in some cases 4 to keep the padding consistent not using rems cause I don't want the padding to increase. It should be the same personnel preference. Thanks for the review and observation.

**Comment by Aayushjain-code** [comment]
Great UI Yash keep working on it!

**Review by vrx29 — APPROVED** [review]
UI looks so clean and awesome great work man.

File: styles/pages/docs.css (hunk 1 above)
Comment (hsnice16) [reply styles/pages/docs.css:27]: Oh! nice then.

**Comment by vashnavichauhan1825** [comment]
your code is so clean and easy to read ! everything is fine:)

**Pull request merged**